SSI: Update corosync config not choosing pacemaker as dep

ref: https://build.whamcloud.com/job/integration-tests-shared-storage-configuration/736/arch=x86_64,distro=el7,label=IML/consoleFull

We have a test where we change the mcast port on a server here:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/a3e71b2a6b76e86d2706fc26a3f7ea2ef2589bce/tests/integration/shared_storage_configuration/test_corosync.py#L159-L190

The command that's executed is supposed to stop pacemaker and corosync, change the mcast port, and restart. It should look like this:

<img width="616" alt="screen shot 2018-11-29 at 3 38 00 pm" src="https://user-images.githubusercontent.com/458717/49250662-5c87c300-f3ed-11e8-8d4a-c99f9e8c5fae.png">


However when the test fails it looks like this:

<img width="612" alt="screen shot 2018-11-29 at 3 37 21 pm" src="https://user-images.githubusercontent.com/458717/49250673-69a4b200-f3ed-11e8-8ad4-3449025d2b43.png">

Because the state machine chooses to only stop / start corosync and pacemaker.service has a `Requires=corosync` in it's unit, pacemaker is stopped but never restarted. This manifests itself as a traceback like:

```
Traceback (most recent call last):
  File "/usr/share/chroma-manager/tests/integration/shared_storage_configuration/test_corosync.py", line 182, in test_mcast_changes
    self.wait_for_assert(lambda: self.assertNoAlerts(server['resource_uri'], of_type='HostOfflineAlert'))
  File "/usr/share/chroma-manager/tests/integration/core/utility_testcase.py", line 180, in wait_for_assert
    "Timed out waiting for %s\nAssertion %s" % (inspect.getsource(lambda_expression), assertion))
AssertionError: Timed out waiting for             self.wait_for_assert(lambda: self.assertNoAlerts(server['resource_uri'], of_type='HostOfflineAlert'))

Assertion u'/api/host/1/' unexpectedly found in [u'/api/host/1/']
```

Need to dig in more to figure out why pacemaker is being omitted, but I wonder if we should just toggle pacemaker.service since it will stop / start corosync anyway due to it's hard dependency.

--- a/tests/integration/shared_storage_configuration/test_corosync.py
+++ b/tests/integration/shared_storage_configuration/test_corosync.py
@@ -25,6 +25,12 @@ class TestCorosync(ChromaIntegrationTestCase):
 
         self.server_configs.extend(self.add_hosts([server['address'] for server in self.TEST_SERVERS[len(self.server_configs):len(self.server_configs) + n_hosts]]))
 
+        def iml_says_pacemakers_running():
+            xs = self.get_list("/api/pacemaker_configuration/")
+            return all([x['state'] == 'started' for x in xs])
+
+        self.wait_until_true(iml_says_pacemakers_running)
+
     # Check that hosts status is updated
     def _host_on_off_line(self, index, online):
         corosync_configuration = self.get_list("/api/corosync_configuration/",

